<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderListDialog</class>
 <widget class="QDialog" name="OrderListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обмен данными лабораторных исследований</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabOrder">
      <attribute name="title">
       <string>Заявки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="chkOrderError">
           <property name="text">
            <string>Текст ошибки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtOrderError">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOrderCount">
         <property name="text">
          <string>Записей в таблице: 0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CTableView" name="tblOrder"/>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnOrderApply">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Дата создания:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkOrderDateFrom">
           <property name="text">
            <string>c</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CDateEdit" name="edtOrderDateFrom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkOrderDateTo">
           <property name="text">
            <string>по</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CDateEdit" name="edtOrderDateTo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOrderStatus">
           <property name="text">
            <string>Статус</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CEnumComboBox" name="cmbOrderStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOrderSyncStatus">
           <property name="text">
            <string>Текущее состояние</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CEnumComboBox" name="cmbOrderSyncStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOrderResponse">
      <attribute name="title">
       <string>Результаты</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOrderResponseCount">
         <property name="text">
          <string>Записей в таблице: 0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="chkOrderResponseDate">
           <property name="text">
            <string>Дата создания</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CDateEdit" name="edtOrderResponseDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOrderResponseStatus">
           <property name="text">
            <string>Статус</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CEnumComboBox" name="cmbOrderResponseStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOrderResponseSyncStatus">
           <property name="text">
            <string>Текущее состояние</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CEnumComboBox" name="cmbOrderResponseSyncStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnOrderResponseApply">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="CTableView" name="tblOrderResponse"/>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="chkOrderResponseError">
           <property name="text">
            <string>Текст ошибки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtOrderResponseError"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>library/Enum.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblOrder</tabstop>
  <tabstop>chkOrderDateFrom</tabstop>
  <tabstop>edtOrderDateFrom</tabstop>
  <tabstop>chkOrderDateTo</tabstop>
  <tabstop>edtOrderDateTo</tabstop>
  <tabstop>cmbOrderStatus</tabstop>
  <tabstop>cmbOrderSyncStatus</tabstop>
  <tabstop>chkOrderError</tabstop>
  <tabstop>edtOrderError</tabstop>
  <tabstop>btnOrderApply</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>tblOrderResponse</tabstop>
  <tabstop>chkOrderResponseDate</tabstop>
  <tabstop>edtOrderResponseDate</tabstop>
  <tabstop>cmbOrderResponseStatus</tabstop>
  <tabstop>cmbOrderResponseSyncStatus</tabstop>
  <tabstop>chkOrderResponseError</tabstop>
  <tabstop>edtOrderResponseError</tabstop>
  <tabstop>btnOrderResponseApply</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkOrderDateFrom</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtOrderDateFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrderResponseDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtOrderResponseDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrderError</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtOrderError</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrderResponseError</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtOrderResponseError</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrderDateTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtOrderDateTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
